src/components/ProductForm/BackButton.js

```diff
@@ -0,0 +1,18 @@
+import { MdNavigateBefore } from 'react-icons/md'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+function BackButton() {
+  return (
+    <Button>
+      <MdNavigateBefore />
+      <Link to="/">Back to Products</Link>
+    </Button>
+  )
+}
+
+const Button = styled.button`
+  font-size: large;
+`
```

Comment:
You can style the `Link` directly instead:
```suggestion
const LinkStyled = styled(Link)`
  font-size: large;
`
```